test/parallel/test-event-emitter-remove-all-listeners.js

```diff
@@ -38,24 +38,25 @@ function expect(expected) {
 
 {
   const ee = new events.EventEmitter();
-  ee.on('foo', common.noop);
-  ee.on('bar', common.noop);
-  ee.on('baz', common.noop);
-  ee.on('baz', common.noop);
+  const noop = common.mustNotCall();
```

Comment:
+1
But give it a better name... maybe `mustNotCall` or if your thight with char `dontCall` or even `doNot`